### ISSUE TITLE
Attempt to enable data capture on an endpoint created by cloudformation will fail

### ISSUE THREAD

**Issue by brightsparc** [aws/sagemaker-python-sdk#1301]
**Describe the bug**
Attempt to enable data capture on an endpoint created by cloudformation will fail due as it contains "aws:cloudformation:*" tags.  These tags can not be deleted.

**To reproduce**
Attempt to enable data capture on an endpoint created with "aws:" tags:

see: [ageMaker-Enable-Model-Monitor](https://github.com/awslabs/amazon-sagemaker-examples/blob/master/sagemaker_model_monitor/enable_model_monitor/SageMaker-Enable-Model-Monitor.ipynb) notebook

```
predictor = RealTimePredictor(endpoint=endpoint_name)
predictor.update_data_capture_config(data_capture_config=data_capture_config)
```

Returns error:

```
~/anaconda3/envs/python3/lib/python3.6/site-packages/botocore/client.py in _make_api_call(self, operation_name, api_params)
    584             error_code = parsed_response.get("Error", {}).get("Code")
    585             error_class = self.exceptions.from_code(error_code)
--> 586             raise error_class(parsed_response, operation_name)
    587         else:
    588             return parsed_response

ClientError: An error occurred (ValidationException) when calling the DeleteTags operation: System prefix 'aws:' reserved for AWS usage in tag key: aws:cloudformation:stack-name
```

**Expected behavior**
The "aws:*" tags should either be ignore or removed when collecting existing tags.  

The following is a work around that overrides the tags instead of loading existing

```
# Get the current endpoint config 
endpoint_config_name = sm.describe_endpoint(EndpointName=endpoint_name)['EndpointConfigName']
new_config_name = name_from_base(base=endpoint_config_name)

# Create a new config from the existing adding data capture
new_tags = [{'Key': 'datacapture', 'Value': 'true'}] 
sm_session.create_endpoint_config_from_existing(
    endpoint_config_name, new_config_name, new_tags=new_tags, 
    new_data_capture_config_dict=data_capture_config._to_request_dict())

# Update the endpoint
sm_session.update_endpoint(endpoint_name=endpoint_name, endpoint_config_name=new_config_name)
```

**System information**
A description of your system. Please provide:
- **SageMaker Python SDK version**: 1.50.9.post0
- **Python version**:  3.6.5  
- **CPU**:
- **Custom Docker image**:N

**Comment by laurenyu** [comment]
sorry for the delay in response here, and thank you for the bug report! I've opened a PR to fix this: #1362

**Comment by laurenyu** [comment]
released in [v1.51.4](https://github.com/aws/sagemaker-python-sdk/releases/tag/v1.51.4)
